data.tf

```diff
@@ -7,8 +7,8 @@ locals {
   aat_cft_vnet_name           = "cft-aat-vnet"
   aat_cft_vnet_resource_group = "cft-aat-network-rg"
 
-  app_aks_network_name    = var.env == "sbox" || var.env == "perftest" || var.env == "ithc" || var.env == "aat" || var.env == "preview" ? "cft-${local.aks_env}-vnet" : "core-${local.aks_env}-vnet"
-  app_aks_network_rg_name = var.env == "sbox" || var.env == "perftest" || var.env == "ithc" || var.env == "aat" || var.env == "preview" ? "cft-${local.aks_env}-network-rg" : "aks-infra-${local.aks_env}-rg"
+  app_aks_network_name    = var.env == "sbox" || var.env == "perftest" || var.env == "ithc" || var.env == "aat" || var.env == "preview" || var.env == "prod" ? "cft-${local.aks_env}-vnet" : "core-${local.aks_env}-vnet"
```

Comment:
I think we do not need conditions anymore, with prod all of them would have "CFT" ??